<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixForm</class>
 <widget class="QWidget" name="MixForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGraphicsView" name="src_g_1">
         <property name="dragMode">
          <enum>QGraphicsView::NoDrag</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="src_p_1"/>
         </item>
         <item>
          <widget class="QPushButton" name="src_b_1">
           <property name="text">
            <string>选择图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGraphicsView" name="src_g_2"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="src_p_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="src_b_2">
           <property name="text">
            <string>选择图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGraphicsView" name="out_g"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>保存路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="out_save_p">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="out_save_b">
         <property name="text">
          <string>自定义</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mix_b">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>合成</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
